### PR TITLE
feat: add default DiskGCThresholdPercent and ignore it when is 0

### PR DESCRIPTION
**Pull request by jim3ma**
Signed-off-by: Jim Ma <majinjing3@gmail.com>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist:

- [ ] Code compiles correctly.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by gaius-qi — APPROVED** [review]
LGTM

File: client/daemon/storage/storage_manager.go [hunk 1]
```diff
@@ -716,6 +716,9 @@ func (s *storageManager) forceGC() (bool, error) {
 }
 
 func (s *storageManager) diskUsageExceed() (exceed bool, bytes int64) {
+	if s.storeOption.DiskGCThresholdPercent <= 0 {
```
Comment (hyy0322) [inline client/daemon/storage/storage_manager.go:719]: if DiskGCThresholdPercent <= 0, diskUsageExceed never return true?

Comment (jim3ma) [reply client/daemon/storage/storage_manager.go:719]: DiskGCThresholdPercent is always great than 0 or equal zero

**Review by hyy0322 — APPROVED** [review]
LGTM

**Pull request merged**